pmm-tests/pmm-framework.sh

```diff
@@ -303,6 +303,10 @@ do
     --dev )
     shift
     dev=1
+    if [[ -z $pmm_server_version ]];then
```

Comment:
have you checked what happens between line 140 to 150 in this script?

Reply:
Yes I have. I think in the transient period where all the command line arguments are being looped through, order in which `--dev `and `--pmm-server-version` is provided would matter. But our main execution starts only after the loop has checked all the options provided so ordering wouldn't hurt at the end.

Reply:
Actually I first placed my edit at the lines you mentioned, but then I realized that `dev-latest` works only with `perconalab` dockerhub repo, and not `percona` repo. So I decided to set the default value at `--dev` option.